draft-ietf-rats-eat.md

```diff
@@ -1102,94 +1102,46 @@ The values for the results enumerated type are as follows:
 
 ### Submodules (submods) {#submods}
 
-Some devices are complex, having many subsystems.  A
-mobile phone is a good example. It may have several connectivity
-subsystems for communications (e.g., Wi-Fi and cellular). It may have
-subsystems for low-power audio and video playback. It may have multiple
-security-oriented subsystems like a TEE and a Secure Element.
+Some devices are complex and have many subsystems.  A mobile phone is a good example. It may have subsystems for communications (e.g., Wi-Fi and cellular), low-power audio and video playback, multiple
+security-oriented subsystems like a TEE and a Secure Element, and etc. The claims for a subsystem can be grouped together in a submodule.
 
-The claims for a subsystem can be grouped together in a submodule or submod.
+A submodule claim is defined as a map, where the map label identifies the submodule. 
```

Comment:
"The submodule claim is defined as a map containing submodules where the map labels name the submodules" (or similar).  "map" occurs twice in that sentence, but two uses are not for the same map.

Reply:
Is the first map in the proposed text referring to the map that is the result of CBOR encoding the CDDL group? If so, I don't think we should make this change as that is covered in the front matter of the Claims section and we don't address that for other claims. The next sentence gets at what Submodule contains. Maybe combining the two sentences like this would be an improvement:

```
A submodule claim is defined as a map, where each map label identifies a submodule and the value is a Claims-Set, Nested-Token or Detached-Submodule-Digest.
```

Reply:
Sorry. I wasn't clear. 

The word "map" is used to refer to the same map both types, the map that holds the submods. 

The problem is with "map label", because you might not know which label it refers to -- when you put a map in a map there's two layers of labeling. Two sentences?
     `A submodule claim is a map holds some number of submodule.`
     `Each submodule is named by its label in the submodule claim map`